open_in_mpv/open_in_mpv.py

```diff
@@ -193,6 +193,10 @@ def real_main(log: TextIO) -> int:
         logger.exception('No URL was given')
         print(json.dumps(dict(message='Missing URL!')))
         return 1
+    if 'https' not in url:
+        logger.exception('The url appears to not have a secure connection.')
+        print(json.dumps(dict(message='Security Exception!')))
```

Comment:
Just put the message in the JSON and remove the logger call. Also, no shouting.

Reply:
Well shouting occurs if the URL; but no problem ill fix that.